service/cpsv2/types.go

```diff
@@ -0,0 +1,211 @@
+package cpsv2
+
+// OutputEnrollments identifies a collection of OutputEnrollmentElement objects
+type OutputEnrollments struct {
+	Enrollments []OutputEnrollmentElement `json:"enrollments"`
+}
+
+// OutputEnrollmentElement displays all the information about the process that your certificate goes through
```

Comment:
Whats the purpose of `Output` keyword if there is no Input object ? Would it not make sense to unify this a little bit ?

Reply:
output is returned from server, if I will later implement creation part there will be body objects which might be different based on exp with other APIs

Reply:
But thats like tryiing to solve something we dont face yet. if we are using Output objects we indicate we have input - which is false in this scenario

Reply:
Api itself has input objects and not to rename later, as I might implement them for TF, I would keep output

Reply:
Ok - makes sense